.github/workflows/app-etterlatte-brev-kafka.yaml

```diff
@@ -14,6 +14,14 @@ on:
   push:
     branches:
       - main
+    paths:
+      - apps/etterlatte-brev-kafka/**
+      - "!apps/etterlatte-brev-kafka/.nais/*"
```

Comment:
Denne skurrar litt for meg, mi erfaring er vel at endringar i nais-yamlane ofte fører til behov for å redeploye, men kopierte berre det som låg under pull_request-lista under.

Reply:
enig, den skurrer litt der.